### ISSUE TITLE
Installed but several things not working

### ISSUE THREAD

**Issue by josephgerth** [Syndrizzle/hotfiles#6]
Nice rice. I needed to refresh my system so I thought I would give this rice a try. Unfortunately, I've run into several issues:

1. Certain elements, such as the menu, freeze and stop working. The clock will freeze at a certain time, none of the buttons (like "vitals") animate or open up.
2. I cannot get Alacritty to work -- at all. It's installed, and when I open it, a window will briefly open up and then immediately disappear.
3. How can I edit the weather and messages on the desktop, such as the misspelled "Welcome"?
4. I downloaded Brave and this runs fine. But I must open it explicitly. When I click on "Web Browser" nothing happens. Yes, I have set Brave as my default browser.

I'm pretty sure I installed everything correctly, and I suppose I could start over and reinstall Arch (what I'm using), but I thought I would inquire here first.

Thanks!

**Comment by josephgerth** [comment]
I can't get kitty to open, either. 

From the tty, after installing kitty, I get this message:
`[glfw error 65544]: X11: The DISPLAY environment variable is missing GLFW initialization failed`
I did manage to get st to open; but it seemed to freeze when getting the logs. 

Also, if I try to start alacritty from the tty, I get:
`thread main panicked at Failed to initialize any backend! Wayland status: NoCompositorListening X11 status: XOpenDisplayFailed /build/.cargo/registry/src/github.com...
note: run with RUST_BACKTRACE=1 environment variable to display a backtrace`

**Comment by josephgerth** [comment]
Managed to open kitty; here's what I got:

- eww -c ~/.config/eww/Bar daemon 
- 2022-06-07T00:46:02.808Z INFO  eww > Initializing Eww server. (/run/user/1000/eww-server_f9511714bb92adfa) Run `eww logs` to see any errors while editing your configuration.  2022-06-07T00:46:02.808Z INFO  eww::server > Loading paths: config-dir: /home/user/.config/eww/Bar, ipc-socket: /run/user/1000/eww-server_f9511714bb92adfa, log-file: /home/user/.cache/eww_f9511714bb92adfa.log

- eww -c ~/.config/eww/Bar logs

┏━━━━━━━━━━━━━━━━━━━━━━━┓
┃Initializing eww daemon┃
┗━━━━━━━━━━━━━━━━━━━━━━━┛
    
 2022-06-07T00:46:02.917Z INFO  eww::ipc_server > IPC server initialized
 2022-06-07T00:57:42.692Z INFO  eww::server     > Shutting down eww daemon...
 2022-06-07T00:57:42.692Z INFO  eww::app        > Received kill command, stopping server!
 2022-06-07T00:57:42.692Z INFO  eww::server     > main application thread finished

That's all I could get from the logs. I waited about 10 minutes.

**Comment by Syndrizzle** [comment]
Did you open another window and run that command as well?

**Comment by josephgerth** [comment]
Progress: I got alacritty to run! Apparently I needed fish; I do not recall seeing that program as required on the list, though.

From alacritty, I typed in the log command again (yes, on a brand new terminal window):

  │
84 │         (scale :class "brightness-scale" :width 100 :active true :value brightness-lvl :min 0 :max 101 :orientation "h" :onchange "brightnessctl s {}%")))))
   │                                                                         ────────────── `` is not of type `f64`
   │
   → cannot parse float from empty string


 2022-06-07T01:29:53.811Z INFO  eww::server     > Shutting down eww daemon...
tint2: remove_icon: 6291477 (parcellite)
 2022-06-07T01:29:53.811Z INFO  eww::app        > Received kill command, stopping server!
